docs/user_guide/build_intel_boot_bin.rst

```diff
@@ -0,0 +1,581 @@
+BootROM runs. It bridges the gap between the limited functionality of the
+BootROM and the end user application, configuring the system as needed before
+the larger application can run
+
+
+Below there're useful links who explain in more detail the process and the
+necessary components:
+
+-  `Booting and Configuration <https://www.intel.com/content/dam/support/us/en/programmable/support-resources/bulk-container/pdfs/literature/hb/arria-v/av-5400a.pdf#:~:text=This%20topic%20describes%20the%20booting%20of>`__
+-  `SocBootFromFPGA <https://community.intel.com/t5/FPGA-Wiki/SocBootFromFPGA/ta-p/735773#:~:text=Introduction.%20The%20HPS%20(Hardened%20Processor%20System)>`__
+-  `Intel Bootloader <https://www.intel.com/content/www/us/en/support/programmable/support-resources/design-guidance/soc-bootloader.html#:~:text=Intel%C2%AE%20SoC%20FPGAs%20use%20a%20bootloader>`__
+
+Necessary files for booting up a HDL project
+-------------------------------------------------------------------------------
+
+.. warning::
+   
+   We don't provide any scripts to automate this whole process, so please read
+   carefully and follow all the steps in order to obtain the desired results.
+
+When booting a system with Linux + HDL (depending on the project), you will need
+to write the following files on the root of the BOOT FAT32 partition. After that,
+you must write the preloader image in a specific card partition, depending on
+the case. 
+
+-  For Arria10 SoC projects:
+
+   -  ``socfpga_arria10-common/zImage``
+   -  ``<target>/fit_spl_fpga.itb``
+   -  ``<target>/socfpga_arria10_socdk_sdmmc.dtb``
+   -  ``<target>/u-boot.img``
+   -  ``<target>/extlinux.conf → extlinux /extlinux.conf``
+   -  ``<target>/u-boot-splx4.sfp`` to the corresponding SD card partition
+      (e.g. ``dd if=preloader_bootloader.bin of=/dev/mmcblk0p3``,
+      usually it is the 3rd partition, which is only a couple of MB in size)
+
+-  For Cyclone5 SoC projects:
+
+   -  ``<target>/soc_system.rbf``
+   -  ``<target>/socfpga.dtb``
+   -  ``<target>/uImage``
+   -  ``<target>/preloader_bootloader.img`` to the corresponding SD card
+      partition (e.g. ``dd if=preloader_bootloader.img of=/dev/mmcblk0p3``)
+
+-  For De10Nano SoC project:
+
+   -  ``socfpga_cyclone5_common/zImage``
+   -  ``<target>/extlinux.conf → extlinux /extlinux.conf``
+   -  ``<target>/u-boot.scr``
+   -  ``<target>/socfpga.dtb``
+   -  ``<target>/soc_system.rbf``
+   -  ``<target>/u-boot-with-spl.sfp`` to the corresponding SD card partition
+      (e.g. ``dd if=u-boot-with-spl.sfp of=/dev/sdd3``)
+
+Examples of building the boot image
+-------------------------------------------------------------------------------
+
+Below is a list of projects supported by us for each carrier to illustrate how
+to build and showcase the different files that are involved in the process. Each
+project has it's own characteristics (some files may be different from a project
+to another).
```

Comment:
```suggestion
This is a list of projects supported by us for each carrier. The purpose is to illustrate how
to build and showcase the different files that are involved in the process. Each
project has its own characteristics (some files may differ from one project
to the other).
```
Maybe the first line I edited can be split on the next one, to fit under the character limit per line (I can't see it here)